src/fsharp/FSharp.Core.Unittests/FSharp.Core/Lazy.fs

```diff
@@ -1,172 +0,0 @@
-// Copyright (c) Microsoft Open Technologies, Inc.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
This file was not included in any active testing. It's duplicated in LazyType.fs, and in any case the type is part of the .NET Fx and tested there